src/deprecated.jl

```diff
@@ -1,3 +1,10 @@
 import Base: @deprecate
 
 @deprecate DataFrame!(args...; kwargs...) DataFrame(args...; copycols=false, kwargs...)
+
+# TODO: remove these definitions in year 2021
+by(args...; kwargs...) = throw(ArgumentError("by function was removed from DataFrames.jl. " *
+                                             "Use the `groupby` and `combine` functions instead."))
```

Comment:
This may be more explicit?
```suggestion
                                             "Use the `combine(groupby(...), ...)` or `combine(f, groupby(...))` instead."))
```